src/main/java/duke/tasklist/TaskList.java

```diff
@@ -0,0 +1,197 @@
+package duke.tasklist;
+
+import java.util.List;
+import java.util.ArrayList;
+
+import duke.DukeException;
+import duke.ui.Ui;
+import duke.task.Task;
+import duke.task.ToDo;
+import duke.task.Deadline;
+import duke.task.Event;
+
+/**
+ * Class for operations to task list.
+ */
+public class TaskList {
+    /**
+     * ArrayList of tasks.
+     */
+    private ArrayList<Task> tasks;
+
+    /**
+     * Default constructor for TaskList. Used in case of unable to locate file.
+     */
+    public TaskList() {
+        tasks = new ArrayList<Task>();
+    }
+
+    /**
+     * Constructor for TaskList.
+     *
+     * @param tasks ArrayList of tasks
+     */
+    public TaskList(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    /**
+     * Carry out operations according to commands from user.
+     *
+     * @param result integer passed from Parser class
+     * @param input  original String input from user
+     * @return ArrayList of tasks
+     */
+    public List<Task> doCommand(int result, String input) {
+        String[] s = input.split("\\s");
+        switch (result) {
+            case 0:
+                return tasks;
+            case 1:
+                System.out.print(Ui.line);
```

Comment:
There seems to be a lot of repeated use of Ui.indent and Ui.line, perhaps you could abstract this out to a separate formatter method?